### PR TITLE
feat: [PL-24216]: Added 31-variables module

### PR DESCRIPTION
**Pull request by meenakshi-harness**
##### Screenshots:
This PR:
1. Adds a new module for variables.
2. Added Variable Icon on Resource Page.
3. Added 2 routes for list and details Page.
4. Some string movement to common module (needed for variable module)

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by meenakshi-harness** [comment]
retrigger coverage

**Comment by meenakshi-harness** [comment]
retrigger coverage

File: src/modules/31-variables/pages/variables/__tests__/VariablesPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import { TestWrapper } from '@common/utils/testUtils'
+import VariablesPage from '../VariablesPage'
+
+jest.useFakeTimers()
+
+describe('Variables Page', () => {
+  test('render page at account level', () => {
+    const { getByText } = render(
+      <TestWrapper path="/account/:accountId/resources/variables" pathParams={{ accountId: 'dummy' }}>
```
Comment (reetika04) [inline src/modules/31-variables/pages/variables/__tests__/VariablesPage.test.tsx:19]: Please migrate to route definitions

**Review by reetika04 — APPROVED** [review]

**Comment by meenakshi-harness** [comment]
retrigger coverage

**Comment by meenakshi-harness** [comment]
retrigger titlecheck

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

**Review by chiragp1212 — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by williamwissemann — APPROVED** [review]

**Pull request merged**